optimum/exporters/onnx/config.py

```diff
@@ -304,7 +304,7 @@ def generate_dummy_inputs_for_validation(self, reference_model_inputs: Dict[str,
         return reference_model_inputs
 
 
-class EncoderDecoderOnnxConfig(OnnxSeq2SeqConfigWithPast):
+class DummyEncoderDecoderOnnxConfig(OnnxSeq2SeqConfigWithPast):
```

Comment:
Why "Dummy"?

Reply:
I want this as the base class all encoder-decoder type models inherit from. do you have other naming suggestions. Maybe `EncoderDecoderBaseOnnxConfig`

Reply:
yes